packages/flex-plugin-scripts/src/prints/devServerSuccessful.ts

```diff
@@ -1,27 +1,42 @@
-import { logger, paths } from 'flex-dev-utils';
+import { logger } from 'flex-dev-utils';
 import { ServiceUrl } from 'flex-dev-utils/dist/urls';
 import { WebpackType } from '../config';
+import { readPluginsJson } from 'flex-dev-utils/dist/fs';
+import { UserInputPlugin } from 'src/scripts/start';
 
 /**
  * Prints the message when dev-server has successfully compiled
  */
-export default (local: ServiceUrl, network: ServiceUrl, type: WebpackType) => {
+export default ((local: ServiceUrl, network: ServiceUrl, type: WebpackType, userInputPlugins: UserInputPlugin[]) => {
   logger.success('Compiled successfully!');
-  const pkgName = logger.colors.bold(paths().app.name);
-  const hasIndex = type === WebpackType.Complete || type === WebpackType.Static;
-  const jsUrl = type === WebpackType.JavaScript ? `${paths().app.name}.js` : '';
-  const appType = hasIndex  ? 'app' : 'JavaScript bundle';
+
+  for (const plugin of userInputPlugins) {
+    const pkgName = logger.colors.bold(plugin.name);
+    const jsUrl = type === WebpackType.Static || type === WebpackType.Complete ? `${plugin.name}.js` : '';
+    const pluginType = plugin.remote ? 'remote' : 'local';
+
+    logger.newline();
+    logger.info(`Your ${pluginType} plugin ${pkgName} Javascript bundle is running in the browser on:`);
```

Comment:
We are not showing remote plugins URL - so only show plugins that are local anyway.